### ISSUE TITLE
Process classes for SLOs and IDOs

### ISSUE THREAD

**Issue by Mttbnchtt** [SDG-InterfaceOntology/sdgio#156]
I have added the file _Process classes for SLOs and IDOs_ to the MARLO folder on google drive. The file contains the label and description of some processes that we are trying to isolate to describe the links between SLOs and IDOs using a process algebra like the one the @pbuttigieg described in issue 214 of information-artifact-ontology/IAO. 

The creation of such process classes engenders some issues, which I describe in the file. 
The three most important types of issues are the following:
1. I do not know where to find information (or whether that information exists) to define the process classes for SLOs.
2. Without access to that information, I do not know how to break down the process classes for SLOs listed in this draft. Such process classes are now overloaded and we need to simplify them.
3. Many IDOs labels contain vague or incomplete specifications. For example, one IDO refers to "other shocks" without saying which ones. Another IDO refers to "mitigation" without specifying mitigation of what.

@pbuttigieg suggested checking GEMET, AGROVOC, and other UN thesauri. I started looking into those, but I need to research more (e.g. neither GEMET nor AGROVOC defines "mitigation"). These thesauri will probably be useful though. Does @marieALaporte have specific suggestions both about the issues in the google document and about where to find the relevant information?

**Comment by Mttbnchtt** [comment]
In the google [sheet](https://docs.google.com/spreadsheets/d/1fCEAX2HnsqRpW7Pi9KZVdy86KcgoCQbQhhUdIkOnGNM/edit#gid=819194612) in the MARLO folder, I tried to work out the process classes for SLOs an their definitions. I also try to do the same for the first IDOs. 

For the IDO, the situation is more difficult, since the MARLO team does not provide an official definition. In my attempt, I tried to address two issues:
1. What does 'resilience' mean?
2. What are the other shocks to which they refer?

There is another issue that I was unable to solve: who are the poors to which they refer? Are urban people with a low salary counted among the poors here? Are the poors only rural people? 

Any suggestion about any problem mentioned here would be great.
